tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php

```diff
@@ -520,22 +566,105 @@ public function testFixUnicode()
         $this->doTest($expected, $input);
     }
 
+    public function testDoesNotAlignPropertyByDefault()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @param  int       $foobar Description
+     * @return int
+     * @throws Exception
+     * @var    FooBar
+     * @type   BarFoo
+     * @property     string    $foo   Hello World
+     */
+EOF;
+
+        $input = <<<'EOF'
+<?php
     /**
+     * @param    int   $foobar   Description
+     * @return  int
+     * @throws Exception
+     * @var       FooBar
+     * @type      BarFoo
+     * @property     string    $foo   Hello World
+     */
+EOF;
+
+        $this->doTest($expected, $input);
+    }
+
+    public function testAlignsProperty()
+    {
+        $this->fixer->configure(array('tags' => array('param', 'property', 'return', 'throws', 'type', 'var')));
+
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @param    int       $foobar Description
+     * @return   int
+     * @throws   Exception
+     * @var      FooBar
+     * @type     BarFoo
+     * @property string    $foo    Hello World
+     */
+EOF;
+
+        $input = <<<'EOF'
+<?php
+    /**
+     * @param    int   $foobar   Description
+     * @return  int
+     * @throws Exception
+     * @var       FooBar
+     * @type      BarFoo
+     * @property     string    $foo   Hello World
+     */
+EOF;
+
+        $this->doTest($expected, $input);
+    }
+
+    public function testDoesNotAlignWithEmptyConfig()
+    {
+        $this->fixer->configure(array('tags' => array()));
```

Comment:
could any of you guys check other configs and show to us how it's treated in different fixers ?